precompiles/xtokens/src/lib.rs

```diff
@@ -126,16 +122,16 @@ where
 		handle: &mut impl PrecompileHandle,
 		currency_address: Address,
 		amount: U256,
-		fee: U256,
+		_fee: U256,
```

Comment:
We could probably add a comment here explaining why `fee` param is not being used anymore.